### ISSUE TITLE
help! package_control couldnot be appropriately installed on ubuntu

### ISSUE THREAD

**Issue by chenxiaolong1237old** [wbond/package_control#982]
I download the newest code.zip on github and have tried extract it  in Packages folder or simply put the zip in Installed Packages folder.
      In both ways it doesn`t work. it seems that  package_control couldnot be appropriately loaded.
The log with debug on is below.

```
DPI scale: 1
startup, version: 3083 linux x64 channel: stable
executable: /home/chenxl/sublime_text_3/sublime_text
working dir: /home/chenxl/sublime_text_3
packages path: /home/chenxl/.config/sublime-text-3/Packages
state path: /home/chenxl/.config/sublime-text-3/Local
zip path: /home/chenxl/sublime_text_3/Packages
zip path: /home/chenxl/.config/sublime-text-3/Installed Packages
ignored_packages: ["Vintage"]
pre session restore time: 0.125585
startup time: 0.274557
first paint time: 0.301407
reloading plugin Default.block
reloading plugin Default.comment
reloading plugin Default.copy_path
reloading plugin Default.delete_word
reloading plugin Default.detect_indentation
reloading plugin Default.duplicate_line
reloading plugin Default.echo
reloading plugin Default.exec
reloading plugin Default.fold
reloading plugin Default.font
reloading plugin Default.goto_line
reloading plugin Default.history_list
reloading plugin Default.indentation
reloading plugin Default.kill_ring
reloading plugin Default.mark
reloading plugin Default.new_templates
reloading plugin Default.open_context_url
reloading plugin Default.open_file_settings
reloading plugin Default.open_in_browser
reloading plugin Default.pane
reloading plugin Default.paragraph
reloading plugin Default.paste_from_history
reloading plugin Default.quick_panel
reloading plugin Default.save_on_focus_lost
reloading plugin Default.scroll
reloading plugin Default.set_unsaved_view_name
reloading plugin Default.side_bar
reloading plugin Default.sort
reloading plugin Default.swap_line
reloading plugin Default.switch_file
reloading plugin Default.symbol
reloading plugin Default.transform
reloading plugin Default.transpose
reloading plugin Default.trim_trailing_white_space
reloading plugin CSS.css_completions
reloading plugin Diff.diff
reloading plugin HTML.encode_html_entities
reloading plugin HTML.html_completions
reloading plugin Package Control.Package Control
Traceback (most recent call last):
  File "/home/chenxl/sublime_text_3/sublime_plugin.py", line 74, in reload_plugin
    m = importlib.import_module(modulename)
  File "./importlib/__init__.py", line 90, in import_module
  File "<frozen importlib._bootstrap>", line 1584, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1565, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1532, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 584, in _check_name_wrapper
  File "<frozen importlib._bootstrap>", line 1022, in load_module
  File "<frozen importlib._bootstrap>", line 1003, in load_module
  File "<frozen importlib._bootstrap>", line 560, in module_for_loader_wrapper
  File "<frozen importlib._bootstrap>", line 868, in _load_module
  File "<frozen importlib._bootstrap>", line 313, in _call_with_frames_removed
  File "/home/chenxl/.config/sublime-text-3/Packages/Package Control/Package Control.py", line 112, in <module>
    from .package_control.commands import *
  File "/home/chenxl/.config/sublime-text-3/Packages/Package Control/package_control/commands/__init__.py", line 5, in <module>
    from .advanced_install_package_command import AdvancedInstallPackageCommand
  File "/home/chenxl/.config/sublime-text-3/Packages/Package Control/package_control/commands/advanced_install_package_command.py", line 10, in <module>
    from ..package_manager import PackageManager
  File "/home/chenxl/.config/sublime-text-3/Packages/Package Control/package_control/package_manager.py", line 44, in <module>
    from . import loader, text
  File "/home/chenxl/.config/sublime-text-3/Packages/Package Control/package_control/loader.py", line 21, in <module>
    from .package_disabler import PackageDisabler
  File "/home/chenxl/.config/sublime-text-3/Packages/Package Control/package_control/package_disabler.py", line 12, in <module>
    from package_control import events
ImportError: No module named 'package_control'
reloading plugin Package Control.bootstrap
Traceback (most recent call last):
  File "/home/chenxl/sublime_text_3/sublime_plugin.py", line 74, in reload_plugin
    m = importlib.import_module(modulename)
  File "./importlib/__init__.py", line 90, in import_module
  File "<frozen importlib._bootstrap>", line 1584, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1565, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1532, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 584, in _check_name_wrapper
  File "<frozen importlib._bootstrap>", line 1022, in load_module
  File "<frozen importlib._bootstrap>", line 1003, in load_module
  File "<frozen importlib._bootstrap>", line 560, in module_for_loader_wrapper
  File "<frozen importlib._bootstrap>", line 868, in _load_module
  File "<frozen importlib._bootstrap>", line 313, in _call_with_frames_removed
  File "/home/chenxl/.config/sublime-text-3/Packages/Package Control/bootstrap.py", line 36, in <module>
    from .package_control.bootstrap import bootstrap_dependency
  File "/home/chenxl/.config/sublime-text-3/Packages/Package Control/package_control/bootstrap.py", line 23, in <module>
    from . import loader
  File "/home/chenxl/.config/sublime-text-3/Packages/Package Control/package_control/loader.py", line 21, in <module>
    from .package_disabler import PackageDisabler
  File "/home/chenxl/.config/sublime-text-3/Packages/Package Control/package_control/package_disabler.py", line 12, in <module>
    from package_control import events
ImportError: No module named 'package_control'
plugins loaded
```

**Comment by wbond** [comment]
Unfortunately the GitHub zip by itself is not compatible with ST because it places all files in a root folder.

I'm not sure why you are seeing that error - perhaps some files are missing? What happens if you restart ST?

**Comment by chenxiaolong1237** [comment]
the ST runs well, i can see the menu item "preferences->package control",but nothing happens when i enter the command.  the log before shows What happens when i restart ST.

**Comment by wbond** [comment]
This should be fixed by 67e075f84b87a2ac159392344c21c0e2d316585d

**Comment by wbond** [comment]
Also, thank you for taking the time to report the issue!
